R/target_market_share.R

```diff
@@ -106,6 +106,14 @@ target_market_share <- function(data,
     return(empty_target_market_share_output())
   }
 
+  if ("production" %in% colnames(scenario)) {
+    warn("The column `production` has been removed from the dataset `scenario`.
+         The columns `tmsr` and `smsp` will be used instead",
+         class = "scenario_production_column_removed")
+    scenario <- subset(scenario, select = -c(production))
```

Comment:
NIT: This is equivalent to `dplyr::select(scenario, -.data$production)`. Since `dplyr` is already a (very) strong dependency of this package, I would recommend to use dplyr verbs wherever possible for consistency.

Reply:
Will do, and good to know, thanks!